pom.xml

```diff
@@ -657,6 +657,11 @@
             </goals>
             <!-- As soon as possible but after required generate-sources phase -->
             <phase>process-sources</phase>
+            <configuration>
+              <links>
+                <link>http://javadoc.jenkins.io/</link>
```

Comment:
this site is not yet SSL enabled :-(

Reply:
File/reference an INFRA ticket @jtnord ?

Reply:
https://issues.jenkins-ci.org/browse/JENKINS-47807

Reply:
Oh you mean about ssl on javadoc.jenkins.io :-)